liota/dcc_comms/mqtt_dcc_comms.py

```diff
@@ -74,18 +74,19 @@ def __init__(self, edge_system_name, url, port, identity=None, tls_conf=None, qo
 
         self.client_id = client_id
 
+        if self.client_id is None:
+            #  local_uuid generated will be the client ID
+            self.client_id = systemUUID().get_uuid(edge_system_name)
+            log.debug("Auto-Generated local uuid will be the client ID")
+        else:
+            log.debug("Client ID is provided by user")
+
         if mqtt_msg_attr is None:
             #  pub-topic and sub-topic will be auto-generated
-            log.info("pub-topic and sub-topic is auto-generated")
+            log.debug("Pub-topic and Sub-topic are auto-generated")
             self.msg_attr = MqttMessagingAttributes(edge_system_name)
-            if self.client_id is None or self.client_id == "":
-                #  local_uuid generated will be the client ID
-                self.client_id = systemUUID().get_uuid(edge_system_name)
-                log.info("generated local uuid will be the client ID")
-            else:
-                log.info("Client ID is provided by user")
         elif isinstance(mqtt_msg_attr, MqttMessagingAttributes):
-            log.info("User configured pub-topic and sub-topic")
+            log.debug("User configured pub-topic and sub-topic")
```

Comment:
Should we log the topics? I think so if we're going to log this at all.

Reply:
It is logged in MqttMessagingAttributes